thirdparty/DiskANN/include/pq_table.h

```diff
@@ -110,7 +111,7 @@ namespace diskann {
         rearrangement[d] = d;
       chunk_offsets = new uint32_t[num_chunks + 1];
       for (uint32_t d = 0; d <= num_chunks; d++)
-        chunk_offsets[d] = (_u32)(std::min)(ndims, d * chunk_size);
+        chunk_offsets[d] = (_u32) (std::min)(ndims, d * chunk_size);
```

Comment:
I think this line will break lint check

Reply:
From the CI, the lint is fine. But we should not change the original format of the 3rd party code. This must be accidentally changed by VSCode formatting, will change it back.